File: videojuegoDeChinpokomon/src/Ataque/RayoVeloz.java
package Ataque;

import Chimpokomon.Chimpokomon;

public class RayoVeloz extends Ataque {

    public RayoVeloz(Integer dañoValor, String nombre) {
        super(dañoValor, nombre);
        dañoValor = 3;
        nombre = "RayoVeloz";
    }

    @Override
    public void dañar(Chimpokomon chipoAtacante, Chimpokomon chipoEnemigo) {

        if (chipoAtacante.tieneVentajaSobre(chipoEnemigo)) {
            chipoEnemigo.restarVida(4);

        } else {
            chipoEnemigo.restarVida(3);

        }
    }

}

File: ChimpokomonPython/ModuloSRC/Ataque/RayoVeloz.py
from Ataque import Ataque

class RayoVeloz(Ataque):
    def __init__(self):
            Ataque.__init__(self, 3, "RayoVeloz")

    def dañar(self,  chimpokomonAtacante, chimpokomonEnemigo):
        if(chimpokomonAtacante.tieneVentajaSobre(chimpokomonEnemigo)):
            chimpokomonEnemigo.restarVida(4)
        else:
            chimpokomonEnemigo.restarVida(3)


